app/views/admin/home.py

```diff
@@ -181,7 +181,7 @@ def forbidden_view(self):
     def browse_view(self):
         params = request.args.items()
         params = dict((k, v) for k, v in params if v)
-
+        print params
```

Comment:
Please remove necessary print statements that were used while debugging before making a PR :)